### PR TITLE
Issue #2099 - Added code to handle null key serialization for maps

### PR DESCRIPTION
**Pull request by navidsh**
PR for issue #2099:
- Added a null key serializer to handle when maps have `null` keys

### CODE REVIEW

**Comment by leochr** [comment]
please hold off merging this until PR #44 is merged and the commit is pulled into mpOpenAPI-1.0 branch (for the pending PR #2072 in open-liberty)

**Review by arturdzm — APPROVED** [review]
LGTM

**Pull request merged**
